### PR TITLE
Code Organization

### PR DESCRIPTION
**Pull request by nodh**
Bump to 2.0.0 due to extensive code changes:

- Split out implementation of protocols into `vclib-aries` and `vclib-openid` artifacts
- Implement [OpenID 4 Verifiable Credential Issuance](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html)
- Remove notion of "attribute names" and use "attribute types" only

### CODE REVIEW

File: vclib-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/oidvci/IssuerService.kt [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        )
+        return IssuerMetadata(
+            issuer = publicContext,
+            credentialIssuer = publicContext,
+            authorizationServer = "https://eid.egiz.gv.at/",
+            authorizationEndpointUrl = "$publicContext/authorize",
+            tokenEndpointUrl = "$publicContext/token",
+            credentialEndpointUrl = "$publicContext/credential",
+            supportedCredentialFormat = arrayOf(credentialFormat),
+            displayProperties = arrayOf(DisplayProperties(name = "ID Austria Credential", locale = "de"))
```
Comment (JesusMcCloud) [inline vclib-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/oidvci/IssuerService.kt:39]: overly specific hardcoded values?

**Comment by JesusMcCloud** [comment]
I'll try to figure out how common `implementation` dependencies can be declared only once, but still shared.
Looks very cleanly separated otherwise!

**Comment by JesusMcCloud** [comment]
Erros made me realise, that there are case Diffecences between the modules ("KMM", vs. "Kmm")

**Comment by JesusMcCloud** [comment]
> Errors made me realise, that there are case differences between the modules ("KMM", vs. "Kmm")

still open. to be more precise and less confusing, we still have:
 * VcLibAries`Kmm`
 * VcLibOpenId`Kmm`
 * VcLib`KMM`

Is that intentional?

**Comment by nodh** [comment]
In my opinion, `Kmm` looks cleaner, so we should change it.

File: vclib-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/oidc/OpenIdConstants.kt [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+    const val ID_TOKEN = "idToken"
+
+    const val GRANT_TYPE_CODE = "code"
+
+    const val TOKEN_PREFIX_BEARER = "Bearer "
+
+    const val TOKEN_TYPE_BEARER = "bearer"
+
+    const val PATH_WELL_KNOWN_CREDENTIAL_ISSUER = "/.well-known/openid-credential-issuer"
```
Comment (JesusMcCloud) [inline vclib-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/oidc/OpenIdConstants.kt:13]: :+1:

**Pull request merged**